src/extension.ts

```diff
@@ -1710,7 +1710,50 @@ export async function saveFile(doc: vscode.TextDocument, datasetProvider: IZoweT
     }
     const start = path.join(DS_DIR + path.sep).length;
     const ending = doc.fileName.substring(start);
-    const sesName = ending.substring(0, ending.indexOf(path.sep));
+    let sesName = ending.substring(0, ending.indexOf(path.sep));
+
+    // Added special handling for attempt to save favourite node
+    if (sesName === "Favorites") {
```

Comment:
I think "Favorites" should be a global constant rather than hardcoded. However when created it's localized which is compounding the situation and potential error. Suggest we should address this oversight in USSTree and DatasetTree

Reply:
@Colin-Stone Good idea, I will move it.